--- origsrc/gnome-control-center-2.24.0.1/libwindow-settings/Makefile.am	2008-09-24 11:36:21.000000000 -0500
+++ src/gnome-control-center-2.24.0.1/libwindow-settings/Makefile.am	2008-12-15 21:11:25.572000000 -0600
@@ -37,7 +37,7 @@
 	gnome-window-settings-2.0.pc.in		\
 	ChangeLog
 
-wms_flags = -export_dynamic -avoid-version -module
+wms_flags = -export-dynamic -avoid-version -module -no-undefined
 wmsdir = $(WM_MODULE_DIR)
 
 wms_LTLIBRARIES =			\
