--- origsrc/gnome-control-center-3.0.1.1/configure.ac	2011-04-26 19:57:57.000000000 -0500
+++ src/gnome-control-center-3.0.1.1/configure.ac	2011-05-23 18:29:17.058641700 -0500
@@ -113,12 +114,14 @@ PKG_CHECK_MODULES(SOUND_PANEL, $COMMON_M
                   libcanberra-gtk3 >= $CANBERRA_REQUIRED_VERSION
                   libpulse >= $PA_REQUIRED_VERSION
                   libpulse-mainloop-glib >= $PA_REQUIRED_VERSION
-                  gconf-2.0)
+                  gconf-2.0, [have_pulse=yes], [have_pulse=no])
 PKG_CHECK_MODULES(UNIVERSAL_ACCESS_PANEL, $COMMON_MODULES gconf-2.0)
 PKG_CHECK_MODULES(USER_ACCOUNTS_PANEL, $COMMON_MODULES dbus-glib-1
                   polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION
                   gnome-desktop-3.0
                   gdk-pixbuf-2.0 >= $GDKPIXBUF_REQUIRED_VERSION)
+
+AM_CONDITIONAL(BUILD_SOUND, [test x$have_pulse = xyes])
 
 GDESKTOP_PREFIX=`$PKG_CONFIG --variable prefix gsettings-desktop-schemas`
 AC_SUBST(GDESKTOP_PREFIX)
--- origsrc/gnome-control-center-3.0.1.1/panels/Makefile.am	2011-02-23 17:39:22.000000000 -0600
+++ src/gnome-control-center-3.0.1.1/panels/Makefile.am	2011-05-23 18:29:59.237054200 -0500
@@ -8,12 +8,15 @@ SUBDIRS= \
 	mouse \
 	region \
 	info \
-	sound \
 	keyboard \
 	universal-access \
 	user-accounts \
 	datetime
 
+if BUILD_SOUND
+SUBDIRS += sound
+endif
+
 if BUILD_PRINTERS
 SUBDIRS += printers
 endif
