--- origsrc/gnome-control-center-3.0.1.1/panels/datetime/Makefile.am	2011-03-15 12:02:09.000000000 -0500
+++ src/gnome-control-center-3.0.1.1/panels/datetime/Makefile.am	2011-05-23 17:56:15.614309600 -0500
@@ -90,7 +90,7 @@ test_endianess_CFLAGS = $(DATETIME_PANEL
 
 all-local: check-local
 
-check-local: test-timezone-gfx test-endianess test-timezone
+check-local: $(noinst_PROGRAMS)
 	$(builddir)/test-timezone-gfx $(srcdir)/data
 	$(builddir)/test-endianess
 #	$(builddir)/test-timezone
--- origsrc/gnome-control-center-3.2.2/panels/info/Makefile.am	2011-11-08 14:20:01.000000000 -0600
+++ src/gnome-control-center-3.2.2/panels/info/Makefile.am	2011-11-17 23:57:58.183834900 -0600
@@ -18,7 +18,7 @@ test_hostname_CFLAGS = $(INCLUDES)
 
 all-local: check-local
 
-check-local: test-hostname
+check-local: test-hostname$(EXEEXT)
 	$(builddir)/test-hostname $(srcdir)/hostnames-test.txt > /dev/null
 
 ccpanelsdir = $(PANELS_DIR)
