--- origsrc/gnome-control-center-3.10.3/configure.ac	2014-03-03 05:32:52.000000000 -0600
+++ src/gnome-control-center-3.10.3/configure.ac	2014-03-21 01:03:44.160001300 -0500
@@ -117,17 +117,13 @@ LIBGD_INIT([_view-common notification ma
 
 PKG_CHECK_MODULES(LIBLANGUAGE, $COMMON_MODULES gnome-desktop-3.0 fontconfig)
 PKG_CHECK_MODULES(LIBSHORTCUTS, $COMMON_MODULES x11)
-PKG_CHECK_MODULES(SHELL, $COMMON_MODULES x11 polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION)
+PKG_CHECK_MODULES(SHELL, $COMMON_MODULES x11)
 PKG_CHECK_MODULES(BACKGROUND_PANEL, $COMMON_MODULES libxml-2.0 gnome-desktop-3.0
                   gdk-pixbuf-2.0 >= $GDKPIXBUF_REQUIRED_VERSION)
 PKG_CHECK_MODULES(DATETIME_PANEL, $COMMON_MODULES
                   gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION
-                  polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION
                   gdk-pixbuf-2.0 >= $GDKPIXBUF_REQUIRED_VERSION)
-PKG_CHECK_MODULES(DISPLAY_PANEL, $COMMON_MODULES gnome-desktop-3.0 >= 3.1.0
-                  upower-glib >= 0.9.1)
-PKG_CHECK_MODULES(INFO_PANEL, $COMMON_MODULES libgtop-2.0 gl
-		  polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION)
+PKG_CHECK_MODULES(INFO_PANEL, $COMMON_MODULES libgtop-2.0 gl)
 PKG_CHECK_MODULES(KEYBOARD_PANEL, $COMMON_MODULES
                   gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION
                   x11)
@@ -139,18 +135,9 @@ PKG_CHECK_MODULES(MOUSE_PANEL, $COMMON_M
 PKG_CHECK_MODULES(ONLINE_ACCOUNTS_PANEL, $COMMON_MODULES goa-1.0 goa-backend-1.0 >= $GOA_REQUIRED_VERSION)
 PKG_CHECK_MODULES(POWER_PANEL, $COMMON_MODULES upower-glib >= 0.9.1
                   gnome-settings-daemon >= $GSD_REQUIRED_VERSION)
-PKG_CHECK_MODULES(COLOR_PANEL, $COMMON_MODULES
-                  colord >= $COLORD_REQUIRED_VERSION
-                  colord-gtk >= $COLORD_GTK_REQUIRED_VERSION
-                  libsoup-2.4
-                  gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION)
-PKG_CHECK_MODULES(PRINTERS_PANEL, $COMMON_MODULES
-                  polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION)
 PKG_CHECK_MODULES(PRIVACY_PANEL, $COMMON_MODULES)
 PKG_CHECK_MODULES(REGION_PANEL, $COMMON_MODULES
-                  polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION
                   gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION
-                  accountsservice >= $ACCOUNTSSERVICE_REQUIRED_VERSION
                   $IBUS_MODULE)
 PKG_CHECK_MODULES(SCREEN_PANEL, $COMMON_MODULES)
 PKG_CHECK_MODULES(SEARCH_PANEL, $COMMON_MODULES)
@@ -159,12 +144,6 @@ PKG_CHECK_MODULES(SOUND_PANEL, $COMMON_M
                   libpulse >= $PA_REQUIRED_VERSION
                   libpulse-mainloop-glib >= $PA_REQUIRED_VERSION)
 PKG_CHECK_MODULES(UNIVERSAL_ACCESS_PANEL, $COMMON_MODULES)
-PKG_CHECK_MODULES(USER_ACCOUNTS_PANEL, $COMMON_MODULES
-                  polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION
-                  gnome-desktop-3.0
-                  gdk-pixbuf-2.0 >= $GDKPIXBUF_REQUIRED_VERSION
-                  pwquality >= $PWQUALITY_REQUIRED_VERSION
-                  accountsservice >= $ACCOUNTSSERVICE_REQUIRED_VERSION)
 PKG_CHECK_MODULES(SHARING_PANEL, $COMMON_MODULES)
 PKG_CHECK_MODULES(REMOTE_LOGIN_HELPER, glib-2.0 >= $GLIB_REQUIRED_VERSION gio-2.0)
 
--- origsrc/gnome-control-center-3.10.3/panels/Makefile.am	2014-03-21 01:04:17.677918400 -0500
+++ src/gnome-control-center-3.10.3/panels/Makefile.am	2014-03-21 01:01:24.000000000 -0500
@@ -2,8 +1,6 @@
 	common \
 	background \
 	power \
-	color \
-	display \
 	mouse \
 	notifications \
 	online-accounts \
@@ -12,7 +9,6 @@ SUBDIRS= \
 	sound \
 	keyboard \
 	universal-access \
-	user-accounts \
 	datetime \
 	search \
 	privacy \
--- origsrc/gnome-control-center-3.10.3/panels/region/cc-region-panel.c	2014-03-21 01:04:17.683918800 -0500
+++ src/gnome-control-center-3.10.3/panels/region/cc-region-panel.c	2014-03-21 01:01:24.000000000 -0500
@@ -45,7 +45,9 @@
 #include "cc-ibus-utils.h"
 #endif
 
+#ifndef __CYGWIN__
 #include <act/act.h>
+#endif
 
 #include <libgd/gd-notification.h>
 
@@ -95,8 +97,10 @@ struct _CcRegionPanelPrivate {
         GtkListBoxRow *formats_row;
         GtkWidget     *formats_label;
 
+#ifndef __CYGWIN__
         ActUserManager *user_manager;
         ActUser        *user;
+#endif
         GSettings      *locale_settings;
 
         gchar *language;
@@ -132,6 +136,7 @@ cc_region_panel_finalize (GObject *objec
         g_cancellable_cancel (priv->cancellable);
         g_clear_object (&priv->cancellable);
 
+#ifndef __CYGWIN__
         if (priv->user_manager) {
                 g_signal_handlers_disconnect_by_data (priv->user_manager, self);
                 priv->user_manager = NULL;
@@ -141,6 +146,7 @@ cc_region_panel_finalize (GObject *objec
                 g_signal_handlers_disconnect_by_data (priv->user, self);
                 priv->user = NULL;
         }
+#endif
 
         g_clear_object (&priv->permission);
         g_clear_object (&priv->localed);
@@ -383,7 +389,9 @@ update_language (CcRegionPanel *self,
         } else {
                 if (g_strcmp0 (language, priv->language) == 0)
                         return;
+#ifndef __CYGWIN__
                 act_user_set_language (priv->user, language);
+#endif
                 if (priv->login_auto_apply)
                         set_system_language (self, language);
                 maybe_notify (self, LC_MESSAGES, language);
@@ -580,9 +588,11 @@ update_language_from_user (CcRegionPanel
 	CcRegionPanelPrivate *priv = self->priv;
         const gchar *language;
 
+#ifndef __CYGWIN__
         if (act_user_is_loaded (priv->user))
                 language = act_user_get_language (priv->user);
         else
+#endif
                 language = "en_US.utf-8";
 
         g_free (priv->language);
@@ -596,11 +606,13 @@ setup_language_section (CcRegionPanel *s
 	CcRegionPanelPrivate *priv = self->priv;
         GtkWidget *widget;
 
+#ifndef __CYGWIN__
         priv->user = act_user_manager_get_user_by_id (priv->user_manager, getuid ());
         g_signal_connect_swapped (priv->user, "notify::language",
                                   G_CALLBACK (update_language_from_user), self);
         g_signal_connect_swapped (priv->user, "notify::is-loaded",
                                   G_CALLBACK (update_language_from_user), self);
+#endif
 
         priv->locale_settings = g_settings_new (GNOME_SYSTEM_LOCALE_DIR);
         g_signal_connect_swapped (priv->locale_settings, "changed::" KEY_REGION,
@@ -1676,6 +1688,9 @@ login_changed (CcRegionPanel *self)
 static void
 set_login_button_visibility (CcRegionPanel *self)
 {
+#ifdef __CYGWIN__
+        return;
+#else
         CcRegionPanelPrivate *priv = self->priv;
         gboolean has_multiple_users;
         gboolean loaded;
@@ -1690,6 +1705,7 @@ set_login_button_visibility (CcRegionPan
         gtk_widget_set_visible (priv->login_button, !priv->login_auto_apply);
 
         g_signal_handlers_disconnect_by_func (priv->user_manager, set_login_button_visibility, self);
+#endif
 }
 
 static void
@@ -1722,12 +1738,14 @@ setup_login_button (CcRegionPanel *self)
         g_signal_connect_swapped (priv->login_button, "notify::active",
                                   G_CALLBACK (login_changed), self);
 
+#ifndef __CYGWIN__
         g_object_get (priv->user_manager, "is-loaded", &loaded, NULL);
         if (loaded)
                 set_login_button_visibility (self);
         else
                 g_signal_connect_swapped (priv->user_manager, "notify::is-loaded",
                                           G_CALLBACK (set_login_button_visibility), self);
+#endif
 }
 
 static void
@@ -1771,7 +1789,9 @@ cc_region_panel_init (CcRegionPanel *sel
 		return;
 	}
 
+#ifndef __CYGWIN__
         priv->user_manager = act_user_manager_get_default ();
+#endif
 
         priv->cancellable = g_cancellable_new ();
 
--- origsrc/gnome-control-center-3.10.3/polkit/polkit.h	1969-12-31 18:00:00.000000000 -0600
+++ src/gnome-control-center-3.10.3/polkit/polkit.h	2014-03-21 01:01:24.000000000 -0500
@@ -0,0 +1 @@
+#define polkit_permission_new_sync(a,b,c,d) NULL
--- origsrc/gnome-control-center-3.10.3/shell/Makefile.am	2014-03-21 01:04:17.689919100 -0500
+++ src/gnome-control-center-3.10.3/shell/Makefile.am	2014-03-21 01:01:24.000000000 -0500
@@ -52,22 +52,20 @@ gnome_control_center_LDADD =						\
 	$(top_builddir)/libgd/libgd.la					\
 	$(top_builddir)/panels/common/liblanguage.la			\
 	$(top_builddir)/panels/background/libbackground.la		\
-	$(top_builddir)/panels/color/libcolor.la			\
 	$(top_builddir)/panels/datetime/libdate_time.la			\
-	$(top_builddir)/panels/display/libdisplay.la			\
 	$(top_builddir)/panels/info/libinfo.la				\
 	$(top_builddir)/panels/keyboard/libkeyboard.la			\
 	$(top_builddir)/panels/mouse/libmouse-properties.la		\
 	$(top_builddir)/panels/notifications/libnotifications.la	\
 	$(top_builddir)/panels/online-accounts/libonline-accounts.la	\
 	$(top_builddir)/panels/power/libpower.la			\
 	$(top_builddir)/panels/privacy/libprivacy.la			\
 	$(top_builddir)/panels/region/libregion.la			\
 	$(top_builddir)/panels/search/libsearch.la			\
 	$(top_builddir)/panels/sharing/libsharing.la			\
 	$(top_builddir)/panels/sound/libsound.la			\
 	$(top_builddir)/panels/universal-access/libuniversal-access.la	\
-	$(top_builddir)/panels/user-accounts/libuser-accounts.la
+	$(NULL)
 
 if BUILD_WACOM
 gnome_control_center_LDADD += $(top_builddir)/panels/wacom/libwacom-properties.la
--- origsrc/gnome-control-center-3.10.3/shell/cc-panel-loader.c	2014-03-21 01:04:17.698919600 -0500
+++ src/gnome-control-center-3.10.3/shell/cc-panel-loader.c	2014-03-21 01:01:24.000000000 -0500
@@ -33,9 +33,13 @@ extern GType cc_background_panel_get_typ
 #ifdef BUILD_BLUETOOTH
 extern GType cc_bluetooth_panel_get_type (void);
 #endif /* BUILD_BLUETOOTH */
+#ifndef __CYGWIN__
 extern GType cc_color_panel_get_type (void);
+#endif
 extern GType cc_date_time_panel_get_type (void);
+#ifndef __CYGWIN__
 extern GType cc_display_panel_get_type (void);
+#endif
 extern GType cc_info_panel_get_type (void);
 extern GType cc_keyboard_panel_get_type (void);
 extern GType cc_mouse_panel_get_type (void);
@@ -54,7 +58,9 @@ extern GType cc_search_panel_get_type (v
 extern GType cc_sharing_panel_get_type (void);
 extern GType cc_sound_panel_get_type (void);
 extern GType cc_ua_panel_get_type (void);
+#ifndef __CYGWIN__
 extern GType cc_user_panel_get_type (void);
+#endif
 #ifdef BUILD_WACOM
 extern GType cc_wacom_panel_get_type (void);
 #endif /* BUILD_WACOM */
@@ -77,9 +83,13 @@ static struct {
 #ifdef BUILD_BLUETOOTH
   PANEL_TYPE("bluetooth",        cc_bluetooth_panel_get_type    ),
 #endif
+#ifndef __CYGWIN__
   PANEL_TYPE("color",            cc_color_panel_get_type        ),
+#endif
   PANEL_TYPE("datetime",         cc_date_time_panel_get_type    ),
+#ifndef __CYGWIN__
   PANEL_TYPE("display",          cc_display_panel_get_type      ),
+#endif
   PANEL_TYPE("info",             cc_info_panel_get_type         ),
   PANEL_TYPE("keyboard",         cc_keyboard_panel_get_type     ),
   PANEL_TYPE("mouse",            cc_mouse_panel_get_type        ),
@@ -98,7 +108,9 @@ static struct {
   PANEL_TYPE("sharing",          cc_sharing_panel_get_type      ),
   PANEL_TYPE("sound",            cc_sound_panel_get_type        ),
   PANEL_TYPE("universal-access", cc_ua_panel_get_type           ),
+#ifndef __CYGWIN__
   PANEL_TYPE("user-accounts",    cc_user_panel_get_type         ),
+#endif
 #ifdef BUILD_WACOM
   PANEL_TYPE("wacom",            cc_wacom_panel_get_type        ),
 #endif
