--- origsrc/gnome-control-center-3.14.2/panels/datetime/Makefile.am	2013-01-25 08:42:18.000000000 -0600
+++ src/gnome-control-center-3.14.2/panels/datetime/Makefile.am	2015-01-06 14:01:47.000000000 -0600
@@ -42,7 +42,7 @@ test_endianess_CFLAGS = $(DATETIME_PANEL
 
 all-local: check-local
 
-check-local: test-timezone-gfx test-endianess test-timezone
+check-local: $(noinst_PROGRAMS)
 	$(builddir)/test-timezone-gfx $(srcdir)/data
 	$(builddir)/test-endianess
 #	$(builddir)/test-timezone
--- origsrc/gnome-control-center-3.14.2/shell/Makefile.am	2014-10-29 07:52:07.000000000 -0500
+++ src/gnome-control-center-3.14.2/shell/Makefile.am	2015-01-06 14:01:47.000000000 -0600
@@ -120,7 +120,7 @@ test_hostname_SOURCES = hostname-helper.
 test_hostname_LDADD = $(PANEL_LIBS) $(INFO_PANEL_LIBS)
 
 EXTRA_DIST += hostnames-test.txt
-check-local: test-hostname
+check-local: test-hostname$(EXEEXT)
 	$(builddir)/test-hostname $(srcdir)/hostnames-test.txt > /dev/null
 
 -include $(top_srcdir)/git.mk
