--- origsrc/gnome-control-center-3.8.4.1/panels/datetime/Makefile.am	2013-08-07 16:33:15.839379200 -0500
+++ src/gnome-control-center-3.8.4.1/panels/datetime/Makefile.am	2013-08-07 14:45:55.000000000 -0500
@@ -42,7 +42,7 @@ test_endianess_CFLAGS = $(DATETIME_PANEL
 
 all-local: check-local
 
-check-local: test-timezone-gfx test-endianess test-timezone
+check-local: $(noinst_PROGRAMS)
 	$(builddir)/test-timezone-gfx $(srcdir)/data
 	$(builddir)/test-endianess
 #	$(builddir)/test-timezone
--- origsrc/gnome-control-center-3.8.4.1/shell/Makefile.am	2013-07-24 08:22:24.000000000 -0500
+++ src/gnome-control-center-3.8.4.1/shell/Makefile.am	2013-08-07 16:22:13.913519200 -0500
@@ -111,7 +108,7 @@ test_hostname_LDADD = $(PANEL_LIBS) $(IN
 test_hostname_CFLAGS = $(INCLUDES)
 
 EXTRA_DIST += hostnames-test.txt
-check-local: test-hostname
+check-local: test-hostname$(EXEEXT)
 	$(builddir)/test-hostname $(srcdir)/hostnames-test.txt > /dev/null
 
 -include $(top_srcdir)/git.mk
